src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -671,4 +702,14 @@ private String deleteRequest(String path) throws IOException {
         return doRequest(request);
     }
 
+    @Override
+    public Iterable<SCMFile> getDirectoryContent(BitbucketSCMFile parent) throws IOException, InterruptedException {
+        throw new IOException("List Directory Content not supported for BitBucket Server");
```

Comment:
Why no love for Bitbucket server?

Reply:
It's just I don't have a working instance of BitBucket server to test.